web/docs/troubleshooting.md

```diff
@@ -78,16 +78,18 @@ kubectl logs -n <namespace> <cluster-pod-name> -c plugin-barman-cloud --previous
    If you are using your own certificates without cert-manager, you will need
    to verify the entire certificate chain yourself.
 
+
 2. **Image pull errors**
 
    ```sh
    # Check pod events for image pull errors
-   kubectl describe pod -n cnpg-system -l app.kubernetes.io/name=barman-cloud
+   kubectl describe pod -n cnpg-system -l app=barman-cloud
```

Comment:
Good catch! However, I am not sure we should use a non-qualified label name like `app` (but that's another topic).